streampark-console/streampark-console-service/src/main/assembly/script/schema/mysql-schema.sql

```diff
@@ -191,6 +191,7 @@ create table `t_flink_project` (
   `user_name` varchar(64) collate utf8mb4_general_ci default null,
   `password` varchar(64) collate utf8mb4_general_ci default null,
   `prvkey_path` varchar(128) collate utf8mb4_general_ci default null,
+  `salt` varchar(26) collate utf8mb4_general_ci default null,
```

Comment:
Why is the length 26? I don’t know if it is too short, can you confirm it?

Reply:
The salt length is used in the ShaHashUtils salt generation method, the default length is 26, but the password will be longer after encryption, resulting in database writing problems